documentation/docs/tutorials/native-token-transactions/05-melt-native-tokens.md

```diff
@@ -59,13 +61,13 @@ const nativeTokenOwnerPubKey = "0xa5e76a...";
 const nativeTokenOwnerPrivateKey = "0xc4e210...";
 ```
 
-## Set new state of your Alias Address
+## Set the New State of Your Alias Address
 
-In this step you need to transition the Alias Address to a new state, as one of its bound elements, the Foundry, is changing its state.
+In this step, you need to transition the Alias Address to a new state. As one of its bound elements, the Foundry, is changing its state.
```

Comment:
Are you sure you can start sentence with `As` ?

Reply:
```suggestion
In this step, you need to transition the Alias Address to a new state, as one of its bound elements, the Foundry, is changing its state.
```